modules/core/src/lib/draw-layers.js

```diff
@@ -175,6 +179,19 @@ function drawLayersInViewport(
   const pixelRatio = getPixelRatio({useDevicePixels});
   const glViewport = getGLViewport(gl, {viewport, pixelRatio});
 
+  if (view && view.props.clear) {
```

Comment:
When would the user not want to clear the vewport?

Reply:
> When would the user not want to clear the viewport?

Clearing is not free. We currently clear the entire screen each frame. 

As long as we keep doing that, then when viewports are not overlapping we'd pay the cost twice.

Changing this will require some consideration, since it can lead to some semantic changes. E.g. we allow the user to set background color in `parameters` etc which currently affects the entire screen.

The other case would be more rare, when some overlay effect is intentionally desired.